src/features/ui/MoreNFTOptions/MoreNFTOptions.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from 'react';
+
+import { Option } from './MoreNFTOptions.types';
+import { OptionType } from './MoreNFTOptions.constants';
+
+import { TransferOwnershipModal } from '../../transfer-ownership';
+import { DropdownMenu, DropdownMenuProps } from '@zero-tech/zui/components';
+import { useMoreOptions } from './hooks/useMoreOptions';
+
+type MoreNFTOptionsProps = {
+	domainId: string;
+	domainName: string;
+	domainTitle: string;
+	domainOwner: string;
+	domainCreator: string;
+	trigger: DropdownMenuProps['trigger'];
+};
+
+/**
+ * Wraps the shared functionality of additional NFT options.
+ */
+export const MoreNFTOptions: FC<MoreNFTOptionsProps> = ({
+	domainId,
+	domainName,
+	domainTitle,
+	domainOwner,
+	domainCreator,
+	trigger,
+}) => {
+	const [option, setOption] = useState<Option | undefined>();
+
+	/* Hook returning all options  */
+	const { moreOptions } = useMoreOptions(setOption);
```

Comment:
Will `moreOptions` be used in other components as well? If not, move the hook to this folder.

Reply:
No it will only be required in MoreOptions :) Done.